src/styles/main.scss

```diff
@@ -1,3 +1,245 @@
+$Main-accent: #060b35;
+$Secondary: #616070;
+$Blue-accent: #00acdc;
+
 body {
   margin: 0;
+  line-height: 60px;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: $Main-accent;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  text-transform: uppercase;
+}
+
+.main-navigation {
+  margin-right: 50px;
+
+  &__list {
+    height: 60px;
+    display: flex;
+    margin: 0;
+    line-height: 60px;
+    padding: 0;
+  }
+  &__item {
+    list-style: none;
+    text-transform: uppercase;
+    height: 60px;
+    display: flex;
+    flex-direction: column;
+    color: $Main-accent;
+  }
+
+  &__item:not(:first-of-type) {
+    margin-left: 20px;
+  }
+
+  &__item:first-of-type::after {
+    position: relative;
+    bottom: 4px;
+    content: "";
+    background-color: #00acdc;
+    height: 0;
+    padding: 2px 0;
+    border-radius: 2px;
+    width: 100%;
+  }
+
+  &__item > a {
+    text-decoration: none;
+    color: $Main-accent;
+  }
+
+  &__item > a:hover {
+    color: $Blue-accent;
+  }
```

Comment:
```suggestion
  &__item > a {
    text-decoration: none;
    color: $Main-accent;
    
    &:hover {
       color: $Blue-accent;
    }
  }
```

Reply:
@anpiet  one thing I forgot about - add smooth transitions and animations. Check the readme